MAUI/DataGrid/Column-types.md

```diff
@@ -337,6 +337,56 @@ dataGrid.Columns.Add( new DataGridTextColumn()
 {% endhighlight %}
 {% endtabs %}
 
+### Formatting DataGridTextColumn with different culture
```

Comment:
**Formatting column with different culture**

Move this topic under the DataGridColumn section since this is a common property for all types of columns.

Reply:
Moved